### PR TITLE
update issue template

### PR DESCRIPTION
**Pull request by shiyuhang0**
### What is changed and how it works?
setup an ask a question type for user who have general questions
setup an enhancement type for developer make an enhancement

### CODE REVIEW

**Comment by shiyuhang0** [comment]
@sunxiaoguang @zhangyangyu @birdstorm PTAL

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-name: Bug report
+name: "\U0001F41B Bug Report"
```
Comment (zhangyangyu) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]: If we change the word to title case, please make all the templates same. "Feature Request", "Get Support".

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-name: Feature request
+name: "\U0001F947 Feature request"
```
Comment (birdstorm) [inline .github/ISSUE_TEMPLATE/feature_request.md:2]: Request

Comment (shiyuhang0) [reply .github/ISSUE_TEMPLATE/feature_request.md]: done

**Review by zhangyangyu — APPROVED** [review]

**Pull request merged**